querybook/webapp/components/DataDocScheduleList/DataDocScheduleList.tsx

```diff
@@ -125,6 +126,9 @@ const DataDocScheduleList: React.FC = () => {
                         key={docWithSchedule.doc.id}
                     />
                 ))}
+                {dataDocsWithSchedule.length === 0 && (
+                    <EmptyText>No DataDocs found</EmptyText>
```

Comment:
No Scheduled DataDocs

Reply:
Docs doesn't have to be scheduled to be in this list